_health-care/_js/_components/health-care-app.jsx

```diff
@@ -12,26 +13,51 @@ class HealthCareApp extends React.Component {
   constructor() {
     super();
     this.publishStateChange = this.publishStateChange.bind(this);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
     let initialState = {
       applicationData: {
-        introduction: {},
+        introduction: {
+          introduction: {
+            active: true
```

Comment:
I wasn't sure what the urls should look like for sections. We have separate urls for panels now, but sections don't have any. But you're right I should be making use of the ReactRouter.